dev-java/netty-tcnative/netty-tcnative-1.1.33.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+WANT_AUTOCONF="latest"
+WANT_AUTOMAKE="latest"
+
+inherit autotools autotools-utils eutils java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Tomcat Native fork for Netty"
+
+SLOT="0"
+SRC_URI="https://github.com/netty/${PN}/archive/${PV}.zip"
+HOMEPAGE="http://netty.io/wiki/forked-tomcat-native.html"
+KEYWORDS="~amd64 ~x86"
+LICENSE="Apache-2.0"
+IUSE=""
+
+RDEPEND="dev-libs/apr:1
+	dev-libs/openssl:=
+	>=virtual/jre-1.7"
+
+DEPEND=">=virtual/jdk-1.7
+	${RDEPEND}"
+
+S=${WORKDIR}/${P}/src/main
+
+MY_S="${S}"
+
+JAVA_SRC_DIR="java"
+
+java_prepare() {
+	rm c/os_win32* || die
+
+	cd native-package || die
+
+	mv ../c src || die
+
+	SOURCES=$(ls -w 1000 src | tr "\n" " ") || die
+
+	touch Makefile.am || die
```

Comment:
You say meaningless, one could argue its a QA check to ensure you can create a file before you try to output to it. Yes it can be done with output redirection just the same. Its hardly a critical or major issue, its more a debatable or opinion based way of doing things. Really causes no harm other than some do not like seeing it and would prefer it another way. I would prefer all comments to be in one, and maybe the occasional thank you for your contribution. Working on things others are not. :)